### PR TITLE
BI-4007 service offline landing page

### PR DESCRIPTION
**Pull request by mattch1**
Similar to promise to file - created middleware to load an alternative template if the flag is set.

### CODE REVIEW

File: src/feature.flag.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * Feature flags will be determined by environment variables and all environment variables in nodejs are
+ * either string or undefined. This function will ensure that 'false', '0', 'off' etc remain falsy
+ */
+export default (flag: string | undefined): boolean => {
```
Comment (markpit) [inline src/feature.flag.ts:5]: I was thinking as an improvement, we should give this function a name as when we import it we have to specify a name each time eg. isActiveFeature

**Comment by markpit** [comment]
looks ok but concourse check has failed

**Comment by mattch1** [comment]
I'm not sure how sonar is find this error there is no duplication -seem like a false alarm.

**Comment by markpit** [comment]
> I'm not sure how sonar is finding this error there is no duplication -seems like fake news.

I think you need to merge the latest master into your branch, looks like a possible merge conflict

**Pull request merged**